src/amo/components/SearchForm/styles.scss

```diff
@@ -0,0 +1,80 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+$item-padding-horizontal: 23px;
+$item-padding: 5px $item-padding-horizontal;
+$arrow-width: 18px;
+$addon-icon-margin-end: 12px;
+
+.SearchForm-form {
+  background: $header-search-color;
+  border: 0;
+}
+
+.SearchInput-input,
+.SearchForm-form {
+  color: $base-color;
+  font-size: $font-size-m-smaller;
+  white-space: nowrap;
+}
+
+.SearchForm-suggestions-list {
+  background-color: $white;
+  color: $black;
+  margin: 0;
+  padding: 0;
+}
+
+.SearchForm-suggestions-item {
+  @include text-align-start();
+
+  cursor: pointer;
+  outline: none;
+  padding: $item-padding;
+  width: 100%;
+
+  .Icon-arrow-big-blue {
+    display: none;
+    float: right;
+
+    [dir=rtl] & {
+      float: left;
```

Comment:
I believe we're trying to use flexbox instead of floats everywhere because it's more predictable across browsers but no biggie if it's harder to implement with flex.